codespeed/tests/test_views_data.py

```diff
@@ -173,6 +173,76 @@ def test_get_comparisionexes_branch_filtering(self):
         for index, exe_key in enumerate(expected_exe_keys):
             self.assertEqual(executables[self.project][index]['key'], exe_key)
 
+    def test_get_comparisonexes_tag_name_filtering(self):
+        # Insert some mock revisions with tags
+        rev_v4 = Revision.objects.create(
+            branch=self.branch_master, commitid='4', tag='v4.0.0')
+        rev_v5 = Revision.objects.create(
+            branch=self.branch_master, commitid='5', tag='v5.0.0')
+        rev_v6 = Revision.objects.create(
+            branch=self.branch_master, commitid='6', tag='v6.0.0')
+
+        # No COMPARIION_TAGS filters specified, all the tags should be included
+        executables, exe_keys = getcomparisonexes()
+        self.assertEqual(len(executables), 1)
+        self.assertEqual(len(executables[self.project]), 2 * 2 + 2 * 3)
+        self.assertEqual(len(exe_keys), 2 * 2 + 2 * 3)
+
+        self.assertExecutablesListContainsRevision(executables[self.project], rev_v4)
+        self.assertExecutablesListContainsRevision(executables[self.project], rev_v5)
+        self.assertExecutablesListContainsRevision(executables[self.project], rev_v6)
+
+        # Only a single tag should be included
+        with override_settings(COMPARISON_TAGS=['v4.0.0']):
```

Comment:
This could be three separate unit tests

Reply:
d2f69a92532b1e3c06f7f560518e774f42961ff7